addons/project_issue_sheet/project_issue_sheet.py

```diff
@@ -25,8 +25,30 @@
 class project_issue(osv.osv):
     _inherit = 'project.issue'
     _description = 'project issue'
+
+    def _hours_get(self, cr, uid, ids, field_names, args, context=None):
+        res = {}
+        for issue in self.browse(cr, uid, ids, context=context):
```

Comment:
return {(issue.id, issue.task_id and issue.task_id.progress or 0.0) for issue in self.browse()}